Postgres is up - continuing...
[{"model": "auth.group", "pk": 1, "fields": {"name": "members", "permissions": []}}]